<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="L7MN000108180" name="com.mymatrix.qa.regressiontestcases.ServiceFeeTest" tests="4" failures="0" timestamp="2021-04-01T11:47:43 CDT" time="109.016" errors="1">
  <testcase name="ServiceFeePageTest" time="27.646" classname="com.mymatrix.qa.regressiontestcases.ServiceFeeTest"/>
  <system-out/>
  <testcase name="ServiceFeePageTest" time="27.208" classname="com.mymatrix.qa.regressiontestcases.ServiceFeeTest">
    <error type="java.lang.NumberFormatException" message="For input string: &quot;&quot;">
      <![CDATA[java.lang.NumberFormatException: For input string: ""
at java.lang.NumberFormatException.forInputString(Unknown Source)
at java.lang.Integer.parseInt(Unknown Source)
at java.lang.Integer.parseInt(Unknown Source)
at com.mymatrix.qa.pagemethods.ClientFilePage.navigateToExistingServiceFeeDetailPage(ClientFilePage.java:344)
at com.mymatrix.qa.regressiontestcases.ServiceFeeTest.ServiceFeePageTest(ServiceFeeTest.java:189)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ServiceFeePageTest -->
  <system-out/>
  <testcase name="ServiceFeePageTest" time="26.951" classname="com.mymatrix.qa.regressiontestcases.ServiceFeeTest"/>
  <system-out/>
  <testcase name="ServiceFeePageTest" time="27.211" classname="com.mymatrix.qa.regressiontestcases.ServiceFeeTest"/>
  <system-out/>
</testsuite> <!-- com.mymatrix.qa.regressiontestcases.ServiceFeeTest -->
